sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -323,6 +323,14 @@ class SessionCatalog(
     externalCatalog.databaseExists(dbName)
   }
 
+  def validateGlobalTempView(db: String): Unit = {
```

Comment:
`validateDatabaseName`?

Reply:
I changed to `validateDBIsNotGlobalTempView` which I feel like more close to the code.